packaging/leapp.spec

```diff
@@ -240,7 +244,13 @@ rm -f %{buildroot}/%{_bindir}/leapp
 %dir %{_datadir}/leapp/
 %dir %{_datadir}/leapp/report_schema/
 %{_datadir}/leapp/report_schema
+
+%if %{with python2}
 %{python2_sitelib}/leapp/cli
```

Comment:
define python_leapp_sitelib on the top so you don't have so many conditions around python2_sitelib / python3_sitelib

Reply:
I have been thinking exactly about the same! :D I wanted to discuss planned changes later before I do them as I would like to do even more stuff to make it more simple, but want to be sure everyone is ok with it and I haven't overlooked something.

Reply:
Please propose the changes - I am only looking forward to simplification.

Reply:
done :)